command/core.py

```diff
@@ -29,7 +29,7 @@ def handle_team_join(self, event_data):
         new_id = event_data["event"]["user"]["id"]
         new_user = User(new_id)
         self.__facade.store_user(new_user)
-        welcome = 'Welcome to Lauchpad!'
+        welcome = 'Welcome to Launchpad!'
```

Comment:
Launch Pad 😛

Reply:
Actually, we might even want to say "UBC Launch Pad"... we semi-consistently use the full name.

Reply:
Oops ok